lib/server/ServerRenderer.js

```diff
@@ -72,24 +71,33 @@ function makeClientAppStartScript(clientAppRequirePath, environmentConfig, boots
         "});";
 }
 
-function makeBootstrappingLoader(scriptBody, clientAppUrl, isAsync) {
-    var loadAsync = isAsync ? "s.async = true;\n" : "";
```

Comment:
@zomaish startClientAppAsync is used here. it looks like your changes do not set the script tag to be async. even though we drop the settimeout, the script should still be async, riught?